client/src/app/services/dashboard/dashboard.service.ts

```diff
@@ -19,4 +19,8 @@ export class DashboardService {
   getRecommendations(params: HttpParams): Observable<any> {
     return this.http.get(`${BASEURL}/recommendation/`, { params });
   }
+
+  getInsights(params: HttpParams): Observable<any> {
+    return this.http.get(`${BASEURL}/insights/`, { params });
+  }
```

Comment:
These services should be moved to audits/manage-audits.service.ts

Reply:
fixed